soes/esc.h

```diff
@@ -430,18 +422,15 @@ typedef struct CC_PACKED
    };
 } _FOEh;
 
-#define FOEHSIZE        (sizeof(_FOEh))
-#define FOE_DATA_SIZE   (MBXSIZEBOOT - (MBXHSIZE+FOEHSIZE))
-
 typedef struct CC_PACKED
 {
    _MBXh mbxheader;
    _FOEh foeheader;
    union
    {
-      char filename[FOE_DATA_SIZE];
-      uint8_t data[FOE_DATA_SIZE];
-      char errortext[FOE_DATA_SIZE];
+      char filename;
+      uint8_t data;
+      char errortext;
```

Comment:
This size of this union is now only one byte, is that compensated for elsewhere?